lisa/sut_orchestrator/azure/README.rst

```diff
@@ -0,0 +1,36 @@
+Azure Deployment Template
```

Comment:
Move it to doc folder, and refer it in dev parts.

Reply:
updated